asm/cluster/nodepool.yaml

```diff
@@ -24,8 +24,19 @@ spec:
     minNodeCount: 2
     maxNodeCount: 4 # {"type":"integer","x-kustomize":{"setter":{"name":"max-nodes","value":"4"}}}
   nodeConfig:
+    diskSizeGb: 100
+    diskType: pd-standard
     machineType: e2-standard-4
+    preemptible: false
+    oauthScopes:
+      - "https://www.googleapis.com/auth/cloud-platform"
+    metadata:
+      disable-legacy-endpoints: "true"
+    serviceAccount: anthos-svc-account@PROJECT_ID.iam.gserviceaccount.com # {"type":"string","x-kustomize":{"setter":{"name":"service-account"}}}
     workloadMetadataConfig:
-      nodeMetadata: GKE_METADATA_SERVER
+      nodeMetadata: EXPOSE
```

Comment:
Is this EXPOSE required for anthos sample deployment? At least from ASM side, we've moved away from it.

Reply:
Do you mean the nodeMetadata was EXPOSE and then changed to GKE_METADATA_SERVER in ASM repo? If that's the case, It should be changed to GKE_METADATA_SERVER.

Reply:
correct.

Reply:
done